### ISSUE TITLE
RAM overflow. What do I do?

### ISSUE THREAD

**Issue by AnErrupTion** [kelseyhightower/nocode#4491]
So my code is this:

```

```

however it seems to cause a RAM overflow error like this:

**Memory overflow at address 0x0258741.**

How would I go about fixing this one? I need help ASAP as this is for a school project so yeah.

**Comment by ahpxex** [comment]
This is my solution:
```



```

**Comment by AnErrupTion** [comment]
Your code seems to do something because now I get this error:

**Computer overheating, power supply capacitor explosion in 3 days.**

**Comment by ahpxex** [comment]
> @AnErrupTion 

Maybe you should replace```        ``` to ```                                      ```

**Comment by AnErrupTion** [comment]
Nope, that did not work. Still same error, but now instead of 3 days it's 3 secon-

**Comment by ahpxex** [comment]
This guy's computer killed him, we can take a flower for him
R.I.P.

**Comment by svalordev** [comment]
RIP @AnErrupTion 

```

```

**Comment by AnErrupTion** [comment]
Hello, this is AnErrupTion's parents.

We are here to inform you that he died due to his computer exploding. I hope you persons on the internet can figure out how to not make this happen again for other people to be safe.

Best regards,
AnErrupTion's parents.

**Comment by svalordev** [comment]
I am so sorry for your loss @AnErrupTion's family. I am always just a mention away

**Comment by devkoriel** [comment]
I am facing the same problem. Anybody can help?

**Comment by ahpxex** [comment]
@devkoriel 

You can try the solutions above

**Comment by svalordev** [comment]
RIP @devkoriel The person (@AnErrupTion ) who also had this error is now dead.

**Comment by shaurya-blip** [comment]
Yeah but try @devkoriel 's instruction first before it is too late

**Comment by Hanxiaobo** [comment]
nothing

**Comment by rizkiv1** [comment]
anyone tried downloading more ram?

**Comment by JoannaFalkowska** [comment]
@rizkiv1 Great suggestion, worked perfectly for me.
